plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -151,40 +152,64 @@ fu_plugin_thunderbolt_rescan (FuPlugin *plugin, GError **error)
 		info->id = g_strdup (tdbid);
 		g_ptr_array_add (data->infos, info);
 
-		/* get the vendor ID */
-		rc = tbt_fwu_Controller_getVendorID (data->controllers[i],
-						     &info->vendor_id);
+		rc = tbt_fwu_Controller_isInSafeMode(data->controllers[i], &safe_mode);
```

Comment:
Yes, it's very much like the kernel. In general it's 8-width tabs with braces on the same line, omitted for single lines. I'm not super strict but I think it's important to be self-consistent where we can.